tests/controller_tests/test_ingest.py

```diff
@@ -386,10 +386,11 @@ def test_ingest_endpoint_returns_validation_errors(test_client, example_data_mod
         },
     )
 
+    validation_errors = response.json["validation_errors"]
     assert response.status_code == 400
     assert response.json["detail"] == "Workbook validation failed"
-    assert isinstance(response.json["validation_errors"], list)
-    assert len(response.json["validation_errors"]) == 1
+    assert isinstance(validation_errors, dict)
+    assert "TabErrors" in validation_errors
```

Comment:
We now get two different payloads for both pre-and post-transformation so it might be good to test for both?

Reply:
Good shout added